lib/home_widget.dart

```diff
@@ -50,6 +50,27 @@ class HomeWidget {
     });
   }
 
+  /// Pins the HomeScreen Widget
+  ///
+  /// Android Widgets will look for [qualifiedAndroidName] then [androidName] and then for [name]
+  /// There is no iOS alternative
+  ///
+  /// [qualifiedAndroidName] will use the name as is to find the WidgetProvider
+  /// [androidName] must match the classname of the WidgetProvider, prefixed by the package name
+  static Future<bool?> pinWidget({
```

Comment:
rename to `requestPinWidget` - we are requesting system to pin it, we don't have control over it (low android version, launcher does not support it.